Libraries/LibWeb/CSS/Default.css

```diff
@@ -405,7 +405,7 @@ nobr wbr {
 
 address, blockquote, center, div, figure, figcaption, footer, form, header, hr,
 legend, listing, main, p, plaintext, pre, summary, xmp, article, aside, h1, h2,
-h3, h4, h5, h6, hgroup, nav, section, table, caption, colgroup, col, thead,
+h3, h4, h5, h6, hgroup, nav, section, search, table, caption, colgroup, col, thead,
```

Comment:
Keep this alphabetical please

Reply:
Ah, but consider that this is spec order:
https://html.spec.whatwg.org/#bidi-rendering

( = done, and I've filed [#10780](https://github.com/whatwg/html/pull/10780) over on the html Github)